nncf/openvino/graph/node_utils.py

```diff
@@ -107,17 +107,17 @@ def cnt_if_op(model: ov.Model, cnt: int) -> int:
     return cnt_if_op(model, 0)
 
 
-def get_const_value(const_node: ov.Node, dtype: Optional[np.dtype] = None) -> np.ndarray:
+def get_const_value(const_node: ov.Node) -> np.ndarray:
     """
     Returns the constant tensor for the node.
 
     :param const_node: OpenVINO node.
-    :param dtype: Destination type.
     :return: The constant value.
     """
-    if dtype is None:
-        return const_node.data
-    return const_node.get_data(dtype=dtype)
+    if const_node.get_element_type() == ov.Type.bf16:
+        # Fixed FP32 data type as the result for BF16 constant
```

Comment:
Does this function support other dtypes correctly (like fp16, i16 and etc)?

Reply:
FP16/FP32 - yes. And now - BF16.
Other possible precisions - no, since it is not possible to quantize `int` models or constants yet.

Reply:
Ok, it would be great to have this info in docstring